dvc/remote/local/slow_link_detection.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import time
+
+import colorama
+from dvc.config import Config
+
+logger = logging.getLogger(__name__)
+
+
+class SlowLinkDetectorDecorator(object):
+    LINKING_TIMEOUT_SECONDS = 10.0
+    was_displayed = True
+
+    @classmethod
+    def should_display(cls):
+        if cls.was_displayed:
```

Comment:
Feels like the logic is backwards. Shouldn't `was_displayed` be `False` by default? And when it is displayed then it should set `was_displayed` to `True`? Or am I missing something?